tests/test-class-amp-options-manager.php

```diff
@@ -100,12 +101,12 @@ public function test_get_and_set_options() {
 				'disable_admin_bar'       => false,
 				'all_templates_supported' => true,
 				'supported_templates'     => array( 'is_singular' ),
+				'enable_response_caching' => true,
```

Comment:
The default value here can be the return value of `wp_using_ext_object_cache()`.